Off-Top-App/off-top-python
# OT-81: Create an API in Flask that can post the Session Objects to MongoDB

## What I did:
Created a route that posts a session object to the MongoDB database
Created a Model for the Session object
Created a .gitignore file 

##How to test this branch:
go to the `src` directory run `python app.py`
open up postman switch to a post request and sue this URL: `http://localhost:5000/sessions`
Make an object that looks similar to this:
```
{
	"user_id":"32",
	"first_received_at":"2020-02-29 14:21:41.579",
	"focus_score":"12",
	"transcribed_at":"2020-02-29 14:21:54.477",
	"transcribed_speech":"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Uenim ad minim veniam, quis nostrud"
}
```
![image](https://user-images.githubusercontent.com/28835909/75947755-56855b00-5e56-11ea-9a95-c86100ab53f0.png)



Make sure it returns a list of sessions
